src/System.Drawing.Common/tests/Drawing2D/HatchBrushTests.cs

```diff
@@ -0,0 +1,126 @@
+
+            Assert.NotEqual(backColor, brush.BackgroundColor);
+            Assert.Equal(backColor.ToArgb(), brush.BackgroundColor.ToArgb());
+        }
+
+        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        [InlineData(HatchStyle.Horizontal -1 )]
+        [InlineData(HatchStyle.SolidDiamond + 1)]
+        public void Ctor_InvalidHatchStyle_ThrowsArgumentException(HatchStyle hatchStyle)
+        {
+            Assert.Throws<ArgumentException>(null, () => new HatchBrush(hatchStyle, Color.Empty));
+            Assert.Throws<ArgumentException>(null, () => new HatchBrush(hatchStyle, Color.Empty, Color.Empty));
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void Clone_Brush_ReturnsClone()
+        {
+            var brush = new HatchBrush(HatchStyle.DarkDownwardDiagonal, Color.Magenta, Color.Peru);
+            HatchBrush clone = Assert.IsType<HatchBrush>(brush.Clone());
+
+            Assert.NotSame(clone, brush);
+            Assert.Equal(brush.HatchStyle, clone.HatchStyle);
+            Assert.Equal(brush.ForegroundColor, clone.ForegroundColor);
+            Assert.Equal(brush.BackgroundColor, clone.BackgroundColor);
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void Clone_ImmutableColor_ReturnsMutableClone()
+        {
+            SolidBrush brush = Assert.IsType<SolidBrush>(Brushes.Bisque);
+            SolidBrush clone = Assert.IsType<SolidBrush>(brush.Clone());
+
+            clone.Color = SystemColors.AppWorkspace;
+            Assert.Equal(SystemColors.AppWorkspace, clone.Color);
+            Assert.Equal(Color.Bisque, brush.Color);
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void Clone_Disposed_ThrowsArgumentException()
+        {
+            var brush = new HatchBrush(HatchStyle.DarkHorizontal, Color.PeachPuff, Color.Purple);
+            brush.Dispose();
+
+            Assert.Throws<ArgumentException>(null, () => brush.Clone());
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void HatchStyle_EmptyAndGetDisposed_ThrowsArgumentException()
+        {
+            var brush = new HatchBrush(HatchStyle.DarkHorizontal, Color.PeachPuff, Color.Purple);
+            brush.Dispose();
+
+            Assert.Throws<ArgumentException>(null, () => brush.HatchStyle);
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void ForegroundColor_EmptyAndGetDisposed_ThrowsArgumentException()
+        {
+            var brush = new HatchBrush(HatchStyle.DarkHorizontal, Color.PeachPuff, Color.Purple);
```

Comment:
`using (var brush = new HatchBrush(HatchStyle.DarkHorizontal, Color.PeachPuff, Color.Purple)) { }`
may be preferable style?